libs/MeshKernel/src/CurvilinearGrid/CurvilinearGridFromPolygon.cpp

```diff
@@ -339,10 +339,10 @@ CurvilinearGrid CurvilinearGridFromPolygon::Compute(size_t firstNode,
 
     for (size_t t = 0; t < Mesh::m_numNodesInTriangle; ++t)
     {
-        std::fill(sideOne.begin(), sideOne.end(), Point{constants::missing::doubleValue, constants::missing::doubleValue});
-        std::fill(sideTwo.begin(), sideTwo.end(), Point{constants::missing::doubleValue, constants::missing::doubleValue});
-        std::fill(sideThree.begin(), sideThree.end(), Point{constants::missing::doubleValue, constants::missing::doubleValue});
-        std::fill(sideFour.begin(), sideFour.end(), Point{constants::missing::doubleValue, constants::missing::doubleValue});
+        std::ranges::fill(sideOne, Point{constants::missing::doubleValue, constants::missing::doubleValue});
```

Comment:
We can define a missing point (static Point const MissingPoint(constants::missing::doubleValue, constants::missing::doubleValue) in Entities.hpp for ex.) and use it instead of constructing a Point with missing coords every time it's needed. Less code, better readability. 
This also applies to L. 331-334 above, and possibly other code in the repo.

Reply:
we can use the default constructor

Point() : x(constants::missing::doubleValue), y(constants::missing::doubleValue) {}

Reply:
Even better

Reply:
I'll create an issue